util/web3Storage.js

```diff
@@ -11,13 +11,24 @@ export function makeStorageClient() {
 export function makeFileObjects(data) {
   const buffer = Buffer.from(JSON.stringify(data))
 
-  const files = [
-    new File(['level'], 'plain-utf8.txt'),
-    new File([buffer], 'level.json'),
-  ]
+  const files = [new File([buffer], 'level.json')]
   return files
 }
 
+export async function retrieveFilesData(cid) {
+  const client = makeStorageClient()
+  const res = await client.get(cid)
+  console.log(`Got a response! [${res.status}] ${res.statusText}`)
+  if (!res.ok) {
+    throw new Error(`failed to get ${cid} - [${res.status}] ${res.statusText}`)
+  }
+
+  const files = await res.files()
+  const lvlFile = files[0]
```

Comment:
for now, this is perfect, I guess that later we will have to deal with multiple files and only get the one for the current season if we wanted to keep the files there for history.